bin/proxy.js

```diff
@@ -10,9 +10,11 @@ const cookie = require('cookie');
 const httpProxy = require('http-proxy');
 const pino = require('pino');
 
+const { APP_NAME } = require('../src/core/constants');
+
 const log = pino({
   level: config.get('loggingLevel'),
-  name: `${config.get('appName')}.proxy`,
+  name: `${APP_NAME}.proxy`,
```

Comment:
We could keep `amo` here as well, or any other name you'd like. Even just `proxy` would work. This file is only used for local dev.